06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/src/index.html

```diff
@@ -0,0 +1,121 @@
+        </div>
+    </div>
+    <div id='result'>
+        <!--<div id="user-output">-->
+    <div id= "user-output" ng-show='showIndex' ng-repeat="(filename, filedata) in fileIndices">
+        <p id='paragraph'><b>{{filename}}</b></p>
+        <table>
+            <thead>
+                <tr>
+                    <th>Words</th>
+                    <th ng-repeat='title in titles[filename]'>{{title}}</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr ng-repeat="(key, values) in filedata">
+                    <td> {{key}} </td>
+                    <td ng-repeat="number in docCount[filename]">
+                        <span ng-if="values.includes(number) == true">
+                            <span class="glyphicon glyphicon-ok" aria-hidden="true">√</span>
+                            <!--<span aria-hidden="true"> YES </span>-->
+                        </span>
+                        <span ng-if="values.includes(number) === false">
+                            <span class="glyphicon glyphicon-remove" aria-hidden="true"> X </span>
+                            <!--<span aria-hidden="true"> X </span>-->
+                        </span>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+    <div id= "user-output" ng-show='showSearch' ng-repeat="(filename, filedata) in searchFeedback">
+        <p id='paragraph'><b>{{filename}}</b></p>
+        <table>
+            <thead>
+                <tr>
+                    <th>Words</th>
+                    <th ng-repeat='title in titles[filename]'>{{title}}</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr ng-repeat="(key, values) in filedata">
+                    <td> {{key}} </td>
+                    <td ng-repeat="number in docCount[filename]">
+                        <span ng-if="values.includes(number) == true">
+                            <span class="glyphicon glyphicon-ok" aria-hidden="true">√</span>
+                            <!--<span aria-hidden="true"> YES </span>-->
+                        </span>
+                        <span ng-if="values.includes(number) === false">
+                            <span class="glyphicon glyphicon-remove" aria-hidden="true"> X </span>
+                            <!--<span aria-hidden="true"> X </span>-->
+                        </span>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+    </div>
+    <div id="footer">Copyright &copy; FAM.com</div>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+    <script src="https://code.angularjs.org/1.3.0-rc.1/angular.min.js"></script>
+    <script src="js/inverted-index.js"></script>
+    <script src="js/indexController.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/js/toastr.min.js"></script>
+    </div>
+</body>
+
+</html>
```

Comment:
Yes, I am now leveraging on bootstrap for my sims task and I'll do same for my CP 2.